### PR TITLE
Add X-Content-Type-Options as a custom domain header

### PR DESCRIPTION
**Pull request by ericholscher**
A user requested this via https://infosec.mozilla.org/guidelines/web_security

### CODE REVIEW

File: dockerfiles/nginx/proxito.conf.template [hunk 1]
```diff
@@ -82,6 +82,8 @@ server {
         add_header Access-Control-Allow-Headers $access_control_allow_headers always;
         set $x_frame_options $upstream_http_x_frame_options;
         add_header X-Frame-Options $x_frame_options always;
+        set $x_content_type_options $upstream_http_x_content_type_options;
+        add_header X-Content-Type-Options $x_content_type_options always;
```
Comment (humitos) [inline dockerfiles/nginx/proxito.conf.template:86]: This change probably requires to be duplicated in corporate and also on -ops.

**Review by humitos — APPROVED** [review]
Looks good 👍🏼 

We should have tests for this code eventually 😅

File: dockerfiles/nginx/proxito.conf.template (hunk 1 above)
Comment (ericholscher) [reply dockerfiles/nginx/proxito.conf.template:86]: Hrm... Good point. I'll check before merging.

**Comment by ericholscher** [comment]
Missed this before last deploy 🙃

**Pull request merged**